packages/cli/tests/commonlib/keyVaultValidator.ts

```diff
@@ -22,14 +25,25 @@ const baseUrlVaultSecrets = (vaultBaseUrl: string, secretName: string, secretVer
   `${vaultBaseUrl}/secrets/${secretName}/${secretVersion}?api-version=7.2`;
 const baseUrlVaults = (subscriptionId: string, rg: string, vaultName: string) =>
   `https://management.azure.com/subscriptions/${subscriptionId}/resourceGroups/${rg}/providers/Microsoft.KeyVault/vaults/${vaultName}?api-version=2019-09-01`;
-const baseUrlVaultAddAccessPolicy = (subscriptionId: string, rg: string, vaultName: string) =>
-  `https://management.azure.com/subscriptions/${subscriptionId}/resourceGroups/${rg}/providers/Microsoft.KeyVault/vaults/${vaultName}/accessPolicies/add?api-version=2019-09-01`;
+const baseUrlVaultAddAccessPolicy = (
```

Comment:
minor: better naming to reflect it's purpose? maybe baseUrlVault`Update`AccessPolicy?

Reply:
Cool. Will fix that.

Reply:
Fixed.